book/website/collaboration/inclusive-events.md

```diff
@@ -6,34 +6,40 @@
 - suggestions for how to create an inclusive and accessible agenda
-- accessible technology requirements and reccomendations for hybrid events
+- accessible technology requirements and recommendations for hybrid events
 - and provides a checklist that can be tailored to your specific event
 
 
 (cl-inclusive-events-background)=
 ## Motivation and Background
 
-Every person should have the opportunity to participate fully and equally in all events and social activities – regardless of their individual ability, gender, age, ethnic or social background. 
-This chapter is meant to serve as a resource to support and guide our community members with the tools to identify the opportunities and challenges in upholding this, and to approach event planning and design with accessibility and inclusion in mind.
+Many research projects, especially within the fields of Health and Social Science, are beginning to involve members of the public in the research. This is sometimes referred to as "Public and Patient Involvement and Engagement" or "PPIE" which has a goal to actively involve people with the lived experience of a specific situation, health condition, or experience in the research process. As part of the involvement and engagement, researchers and research staff will need to plan events that are inclusive and accessible for people with lived experience, and delivered in a way that supports people with lived experience taking part equitably. 
 
-Because both internal resources and the events you run can range in size - from from small meetings to massive multi-day conferences - this chapter is not an exhaustive list and instead is meant to provide general guidance about how to design events that are inclusive for people with lived experience. 
+This chapter is meant to serve as a resource to support and guide researchers, organizers, and community members with the tools to identify the challenges and opportunities in organizing events that involve and include people with lived experience.
+
+We refer to "people with lived experience" throughout this chapter as a way to recognize and acknowledge individuals who have direct personal knowledge, or firsthand experience of a particular situation, condition, or topic. 
+It is a term often used to emphasise the importance of including the perspectives and insights of those who have had direct experience of a specific condition or situation. 
+In the context of event planning, the considerations need to cover both the disability requirements of your attendees and an identification of barriers that would be experienced by your specific attendees with lived experience. 
+For example, attendees who are immunocompromised, experience long-term health conditions, people with disabilities, and those with caring responsibilities would all be considered as people with lived experience, and would have slightly different accommodations needed to support their engagement in an event. 
```

Comment:
```suggestion
We refer to "people with lived experience" throughout this chapter as a way to recognize and acknowledge individuals who have direct personal knowledge, or firsthand experience of a particular situation, condition, or topic. It is a term often used to emphasise the importance of including the perspectives and insights of those who have had direct experience of a specific condition or situation. 

Depending on your research, people with lived experience might be people who are immunocompromised, experience long-term health conditions, people with disabilities, or those with caring responsibilities.
In the context of event planning, any of these people with lived experiences would have slightly different accommodation needs to support them engaging in your event. 
Thus, your considerations need to cover both the potential disability requirements of your attendees and an identification of barriers that would be experienced by your specific attendees with lived experience. 

Importantly, not all attendees with lived experience will have a disability, but your event focuses on experiences such as a health condition or disability, then you will need to consider both accessibility and inclusion requirements.
```

@BrainonSilicon I've given it a stab to fold your suggested sentence in and shuffled things around a bit, do you think it makes sense like that? 🙂

Reply:
I think that does! Thank you for the phrasing suggestion 🚀 🚀